docs/contributor/unsorted/debugging-races.md

```diff
@@ -1,9 +1,9 @@
 (debugging-races)=
 <!-- TODO(gfouillet): do not merge into 4.0, or delete whenever merged (reason: related to mongodb) -->
 # Debugging races
-## Triggering races
+## Triggering unit test races
 
-Someone pointed me to the [stress test script](https://github.com/juju/juju/wiki/Stress-Test) on this wiki. While it was
+Someone pointed me to the stress test script in `juju/scripts/unit-test/stress-race.bash`. While it was
```

Comment:
Since we're linking the script directly rather that where it lives, suggest

```
Someone pointed me to the stress test script: `juju/scripts/unit-test/stress-race.bash`.
```

question: Will the script show up as a link in markdown? The encoding looks different.

Reply:
I decided not to make it a link, because the link form was such that, when you clicked on it, it downloaded the script, and that felt wrong.